CHANGELOG.md

```diff
@@ -6,9 +6,142 @@
 
 ## v4.1
 
-### 4.1.0 - Highlights
+### Custom CSS
 
-#### Moved plugins
+Jupyterlab now supports automatic loading of custom CSS.
+Themes are the recommended method for customising the JupyterLab look and feel,
+while custom CSS is intended for minor personal adjustments.
+
+To opt-in start jupyterlab with `--custom-css` flag.
+The location for the `custom.css` is documented the section on [customizing the user interface](https://jupyterlab.readthedocs.io/en/latest/user/interface_customization.html#custom-css).
+
+```{note}
+`custom.css` is also available in Jupyter Notebook;
+while similar selectors can be used for components shared between JupyterLab 4.1 and Jupyter Notebook 7.0,
+a different set of selectors was used in the Notebook 6 and older.
+Users with the custom CSS file using the selectors from legacy Notebook versions
+will need to update them and note that tutorials may refer to the selectors for the old Notebook versions.
+```
+
+### Diagrams in Markdown
+
+Matching GitHub-Flavoured Markdown, a support for diagrams represented with [Mermaid](https://github.com/mermaid-js/mermaid) was added.
+To create a mermaid diagram use the `mermaid` language specifier for a code block in a markdown cell or document, for example:
+
+~~~
+```mermaid
+flowchart LR
+
+A[Hard] -->|Text| B(Round)
+B --> C{Decision}
+C -->|One| D[Result 1]
+C -->|Two| E[Result 2]
+```
+~~~
+
+which renders as:
+
+<img alt="Rendered Mermaid diagram going from left to right with modern look; the diagram contains blueish square node with text Hard pointing to round square node with text Round; over the arrow connecting the two nodes there is a word Text; the second node further connects to a rhombus-shaped node with text Decision which then connects to two further nodes, Result 1 and Result 2." src="https://raw.githubusercontent.com/krassowski/jupyterlab/changelog-4.1/docs/source/getting_started/changelog_assets/4.1-merimad-diagram.png" class="jp-screenshot">
+
+
+### Inline Completer
+
+JupyterLab now supports completion presented as ghost text in the cell and file editors,
+enabling multi-line completions such as those generated by large language models.
```

Comment:
This could be helpful to folks searching for keywords like `copilot`:

```suggestion
enabling multi-line completions such as those generated by large language models (like GitHub Copilot).
```

Reply:
I committed this suggestion but I was a bit on the fence here because even in jupyter-ai we will not interface with GitHub Copilot itself as it is not a publicly documented API (at least was not the last time I looked). Maybe we should use a more generic "GPT-based copilots" which would also avoid touching trademarks?

Reply:
> more generic "GPT-based copilots" which would also avoid touching trademarks?

Sounds good :+1: 

The idea was more to give a sense for what "inline completer" means to the broader crowd, who has very likely heard about GitHub Copilot, or just the term "copilot".